cli/bindata.go

```diff
@@ -14,14 +14,15 @@
 package main
 
 import (
-	"github.com/elazarl/go-bindata-assetfs"
 	"fmt"
+	"github.com/elazarl/go-bindata-assetfs"
```

Comment:
please split system imports from external, this kind of grouping doesn't make sense.